os/arch/arm/src/armv7-a/arm_mmu.c

```diff
@@ -65,6 +65,8 @@ static void mmu_set_flags(uint32_t *val, bool ro, bool exec, uint8_t isL1)
 			*val |= MMU_APP_L1_ROX;
 		} else if (ro) {
 			*val |= MMU_APP_L1_RO;
+		} else if (!ro && exec) {
```

Comment:
I think `!ro` condition is unnecessary because of above `else if (ro)` condition.

Reply:
Fixed.